src/app/docs/key-info/key-info-docs.component.html

```diff
@@ -0,0 +1,164 @@
+<sky-docs-demo-page
+  moduleName="SkyKeyInfoModule"
+  moduleSourceCodePath="src/app/public/modules/key-info/"
+  pageTitle="Key Info"
+>
+  <sky-docs-demo-page-summary>
+    The key info component highlights important summary information.
+  </sky-docs-demo-page-summary>
+
+    <sky-docs-design-guidelines>
+
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+      >
+        <p>
+          Use key info to highlight important summary information.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          caption=" Do use key info to highlight important summary information."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/key-info/key-info-usage-1.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+      >
+        <p>
+          Use key info to show the total number of items in a list. For list totals, use the horizontal layout and place the key info at the top of the list.
```

Comment:
I don' think we need the `For list totals` opening clause in the second sentence since it just restates the preceding sentence. I also think we need a comma before `and` since the second sentence is a compound instruction.